### PR TITLE
Add Set-NsxLogicalRouter function

### PR DESCRIPTION
**Pull request by yktsubo**
Added  Set-NsxLogicalRouter  which is similar to Set-NsxEdge.
This function is discussed in #291

### CODE REVIEW

**Comment by nmbradford** [comment]
Maintainers validate this PR.

**Comment by nmbradford** [comment]
Maintainers validate this PR.

**Comment by nmbradford** [comment]
Maintainers validate this PR.

File: PowerNSX.psm1 [hunk 1]
```diff
@@ -11228,6 +11228,83 @@ function Get-NsxLogicalRouterInterface {
...
+
+    <#
+    .SYNOPSIS
+    Configures an existing NSX Logical Router Raw configuration.
+
+    .DESCRIPTION
+    An NSX Logical Router is a distributed routing function implemented within
+    the ESXi kernel, and optimised for east west routing.
+
+    Use the Set-NsxLogicalRotuer to perform updates to the Raw XML config for an DLR
```
Comment (nmbradford) [inline PowerNSX.psm1]: _a_ DLR :)

**Comment by yktsubo** [comment]
Thank you for your comment. I fixed the grammatical error that you pointed out.
I'm happy to write tests for this and "set-nsxedge".

**Comment by nmbradford** [comment]
awesome - thanks Yuki!  Let me know if you need any advice here...

File: PowerNSX.psm1 (hunk 1 above)
Comment (alagoutte) [inline PowerNSX.psm1]: typo Set-NsxLogicalRotuer

**Comment by nmbradford** [comment]
Jenkins test this please

**Comment by nmbradford** [comment]
Sorry its taken me so long, thanks Yuki!

**Pull request merged**